src/App.js

```diff
@@ -1,28 +1,9 @@
-import logo from './logo.svg';
 import './App.css';
+import Calculator from './components/Calculator';
 
 function App() {
   return (
```

Comment:
- [ ] Good job so far in creating this component and its contents, but it's required you make it a class-based component. Kindly make it a class-based component 👍